data/hooks/conf_regen/15-nginx

```diff
@@ -38,6 +38,15 @@ do_pre_regen() {
   for domain in $domain_list; do
     domain_conf_dir="${nginx_conf_dir}/${domain}.d"
     mkdir -p "$domain_conf_dir"
+    domain_ca_type=$(yunohost domain cert-status ${domain} | grep CA_type | tr -d " ")
+    if [[ "$domain_ca_type" = "CA_type:Let'sEncrypt" ]]; then
+      if [ ! -f "/etc/yunohost/certs/${domain}/chain.pem" ];then
+        wget -O ${pending_dir}/etc/yunohost/certs/${domain}/chain.pem "https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem"
```

Comment:
We need to be careful about this step, because `/etc/yunohost/certs/${domain}` is a symlink that will get updated when we a renew is done.
Hence : 
- either we regen the nginx conf each time we enable a new certificate, to run this piece of code
- or we move this wget to the python code (I'd prefer that solution because it's more efficient that regenerating the whole conf)